java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -556,4 +559,36 @@ public void time_with_route() {
                 "Time() result (" + serverTime[0] + ") should be greater than now (" + now + ")");
         assertTrue(Long.parseLong((String) serverTime[1]) < 1000000);
     }
+
+    @Test
+    @SneakyThrows
+    public void save() {
+        String error = "Background save already in progress";
+        var response = tryCommandWithExpectedError(() -> clusterClient.save(), error);
+        assertTrue(response.getValue() != null || response.getKey().equals(OK));
+
+        var routedResponse = tryCommandWithExpectedError(() -> clusterClient.save(RANDOM), error);
+        assertTrue(
+                routedResponse.getValue() != null || routedResponse.getKey().getSingleValue().equals(OK));
+
+        if (REDIS_VERSION.isGreaterThanOrEqualTo("7.0.0")) {
```

Comment:
Test transactions here https://github.com/Bit-Quill/glide-for-redis/blob/main/java/integTest/src/test/java/glide/cluster/ClusterTransactionTests.java same goes for standalone.